docs/cloud/tcld/account/metrics/enable.md

```diff
@@ -0,0 +1,21 @@
+---
+id: enable
+title: tcld account metrics enable
+sidebar_label: enable
+description: How to enable the metrics endpoint for a Temporal Cloud account using tcld.
+tags:
+  - reference
+  - tcld
+---
+
+The `tcld account metrics enable` command enables the metrics endpoint for the Temporal Cloud account that is currently logged in.
```

Comment:
Both Google and Microsoft are cool with using _enable_ and _disable_ in this limited context. I should take up the question of using _activate_ and _deactivate_ with both the Docs team and the Engineering team. It won't change the hard-coded commands, but it might change how we talk about them.